docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_run_command/_index.md

```diff
@@ -0,0 +1,161 @@
+
+    ![Screenshot Linux Azure Arc-enabled server connected status](./02.png)
+
+    ![Screenshot Windows Azure Arc-enabled server connected status](./03.png)
+
+- [Install or update Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest) on your client machine. Azure CLI should be running version 2.49.0 or later. Use ```az --version``` to check your current installed version.
+- check the version of the Azure cli extension "connectedmachine" on your client machine using the following command
+```shell
+    az extension list --query "[?name=='connectedmachine'].version"
+```
+- If the version of the Azure cli extension is older than "0.5.1" then use the following commands to upgrade it 
+```shell
+    az extension remove --name connectedmachine
+    az extension add -- name connectedmachine
+```
+- Check the version of the Connected Machine agent running on your Arc-connected machine using the following command
+
+```shell
+    az connectedmachine show --name <Machine Name> --resource-group <Resource Group Name> --query "agentVersion"
+```
+- If the Connected Machine agent version is lower than 1.39 then upgrade the extension using [this guidance](https://learn.microsoft.com/en-us/azure/azure-arc/servers/manage-agent).
+
+- To use PowerShell you need to check if the installed version of module _Az.ConnectedMachine_ is 0.6.0 or higher. Use the following PowerShell command to check the installed version
+
+```powershell
+Get-Module -ListAvailable -Name Az.ConnectedMachine
+```
+Use the following command to install the latest version of the PowerShell module
+
+```powershell
+Install-Module -Name Az.ConnectedMachine -Force 
+```
+
+## Use the Run command to execute a simple PowerShell command within an Arc-connected Windows machine
+
+- Run the following Azure cli command after adding the appropriate resource group, name of the Arc-connected machine, a name identifying the command and the location of you Arc-connected machine
+
+```shell
+    az connectedmachine run-command create --resource-group <Resource Group Name> --machine-name <Machine Name> --run-command-name <Identifying Name of command> --script "Write-Host 'Hello World'" --location <Location>
+```
+- It takes few minutes to run the command and return the results. If the execution is successful then you would see the following within the longer returned Json string:
+    
+```shell
+    "executionState": "Succeeded",
+    "exitCode": 0,
+    "output": "Hello World",
+```
+
+If you prefer to use PowerShell then use the following command
+
+```powershell
+New-AzConnectedMachineRunCommand -ResourceGroupName "<Resource Group Name>" -Location "<Location>" -SourceScript "Write-Host 'Hello World'" -RunCommandName "<Identifying Name of command>" -MachineName "<Machine Name>"
+```
+
+The successful output of the PowerShell command will show the following output
+
+![Screenshot success helloworld PowerShell](07.png)
```

Comment:
07 comes before 06.png. Align file names according to it's position in the doc.

Reply:
Fixed the screenshots order in the document